source/installation/java.md

```diff
@@ -55,9 +55,11 @@ Follow the last three steps, except for Structurize (which MineColonies needs to
 
 MineColonies doesn't always accept the latest version of Structurize. To ensure you have the required version, first, click on the MineColonies file name (on the Curseforge website). You'll be taken to the changelog, where it should say the accepted Structurize version at the top. Download that exact version.
 
-#### If Adding Pack, Extract It
+#### If Adding Modpack you will require an external installer.
```

Comment:
Title case here, since it's a heading. Maybe say "Cannot Add Modpack Without External Installer"